### PR TITLE
Mimir cost dashboard

### PR DESCRIPTION
**Pull request by carillan81**
This PR
- add dashboard for prometheus - mimir comparative
image
- add dashboard for mimir cost
image

Follow ups:
- Implement S3 cloudwatch metrics to get storage
- mimir-mixin full research

### Checklist

- [x] Update changelog in CHANGELOG.md in an end-user friendly language.

### CODE REVIEW

File: helm/dashboards/dashboards/shared/private/prometheus-mimir-comparative.json [hunk 1]
```diff
@@ -0,0 +1,751 @@
...
+        },
+        "pluginVersion": "9.5.2",
+        "targets": [
+          {
+            "datasource": {
+              "type": "prometheus",
+              "uid": "PBFA97CFB590B2093"
+            },
+            "editorMode": "code",
+            "expr": "sum(max(\n    container_memory_usage_bytes{container=\"prometheus\", pod=~\"prometheus-($cluster).*\", cluster_type=~\"management_cluster\"}\n) by (container, pod, cluster_type))",
```
Comment (QuantumEnigmaa) [inline helm/dashboards/dashboards/shared/private/prometheus-mimir-comparative.json:165]:
```suggestion
            "expr": "sum(sum(max_over_time(\n    container_memory_usage_bytes{container=\"prometheus\", pod=~\"prometheus-($cluster).*\", cluster_type=~\"management_cluster\"}[$__range]\n)) by (container, pod, cluster_type))",
```

Comment (QuantumEnigmaa) [reply helm/dashboards/dashboards/shared/private/prometheus-mimir-comparative.json:165]: Change you proposed for prometheus in the previous PR :)

**Review by QuantumEnigmaa — CHANGES_REQUESTED** [review]

File: helm/dashboards/dashboards/shared/private/prometheus-mimir-comparative.json (hunk 1 above)
Comment (teemow) [reply helm/dashboards/dashboards/shared/private/prometheus-mimir-comparative.json:165]: Again. This is a sum over a grouped query. The grouping isn't necessary if you sum it anyway.

**Comment by QuantumEnigmaa** [comment]
@carillan81 Any updates ?

**Comment by carillan81** [comment]
I'll check this during the day

File: helm/dashboards/dashboards/shared/private/prometheus-mimir-comparative.json [hunk 2]
```diff
@@ -0,0 +1,751 @@
...
+        },
+        "pluginVersion": "9.5.2",
+        "targets": [
+          {
+            "datasource": {
+              "type": "prometheus",
+              "uid": "PBFA97CFB590B2093"
+            },
+            "editorMode": "code",
+            "expr": "sum(sum(rate(container_cpu_usage_seconds_total{container=\"prometheus\", pod=~\"prometheus-($cluster)-.*\", cluster_type=~\"management_cluster\"}[5m])) by (pod))",
```
[the remark this answers is not in the document]
Comment (carillan81) [reply helm/dashboards/dashboards/shared/private/prometheus-mimir-comparative.json:97]: I think I did this for a reason, but as long as I can't remember it, I guess yopu are right and we don't need to do the aggregation in two steps.

File: helm/dashboards/dashboards/shared/private/prometheus-mimir-comparative.json (hunk 1 above)
Comment (carillan81) [reply helm/dashboards/dashboards/shared/private/prometheus-mimir-comparative.json:165]: both of you are right, changed

**Review by QuantumEnigmaa — APPROVED** [review]

**Pull request merged**